### ISSUE TITLE
Suggestion: Create dynamic buy and sell values for the Store depending on current stock of items.

### ISSUE THREAD

**Issue by RushRepsaJ** [StrangeLoopGames/EcoSuggestions#953]
I would like to be able to make a dynamic store. For example I want to make a warehouse where depending on how much it is filled the price chances. If there is much corn in the warehouse the 'buy' price will be lower compared when there is 0 corn the price will be higher.

This could be done by having multiple buy corn lines in the store, but that is not possible. The most neat way is of course to have some kind of math function in the store itself.

![](https://media.discordapp.net/attachments/281255719372849172/663463924825456660/b395948b669e01cf4e5a13ddbf195afd.png)

Might be related or combined with theses issues if you enable some kind of scripting or logic for store:
https://github.com/StrangeLoopGames/EcoSuggestions/issues/854
https://github.com/StrangeLoopGames/EcoSuggestions/issues/488
https://github.com/StrangeLoopGames/EcoSuggestions/issues/405

**Comment by klhoest** [comment]
I agree, I have to log in often to see my stock and adjust price accordingly.
An alternative could be to be able to change our shop price with the web UI.
